comictaggerlib/issueidentifier.py

```diff
@@ -320,12 +320,15 @@ def _get_issue_cover_match_score(
 
         self._user_canceled()
 
-        urls = [primary_img_url]
-        if use_alt_urls:
-            urls.extend(alt_urls)
-            self.log_msg(f"[{len(alt_urls)} alt. covers]")
+        if isinstance(primary_img_url, ImageHash):
+            remote_hashes = [("0", primary_img_url.Hash)]
```

Comment:
We could also make the alt hashes optionally ImageHashes as well, but this could also be done later